prometrics.go

```diff
@@ -111,6 +111,29 @@ var (
 // GetFactory returns a global-shared CommandFactory, with metrics registered into r
 func GetFactory(r prometheus.Registerer) *CommandFactory {
 	once.Do(func() {
+		initMetrics("")
```

Comment:
Could we just call `GetFactoryWithNameSpace("", r)` inside GetFactory?

Reply:
+1